microsoftteams_connector.py

```diff
@@ -642,14 +685,19 @@ def _generate_new_access_token(self, action_result, data):
         # So we have to check that token from response and token which are saved to state file
         # after successful generation of new token are same or not.
 
-        if (self._access_token != self._state.get("token", {}).get(MSTEAMS_ACCESS_TOKEN_STRING)) or (self._refresh_token != self._state.get
-        ("token", {}).get(MSTEAMS_REFRESH_TOKEN_STRING)):
-            message = "Error occurred while saving the newly generated access or "
-            message += "refresh token (in place of the expired token) in the state file."
-            message += " Please check the owner, owner group, and the permissions of the state file. The Phantom "
-            message += "user should have the correct access rights and "
-            message += "ownership for the corresponding state file (refer to readme file for more information)."
-            return action_result.set_status(phantom.APP_ERROR, message)
+        try:
+            if self._access_token != self.decrypt_state(self._state.get("token", {}).get
```

Comment:
I think there are a couple `"token"` literals here that could be `MSTEAMS_TOKEN_STRING`

Reply:
Updated